### PR TITLE
Add missing Collapsible stories

### PR DESCRIPTION
**Pull request by CloudPower97**
# Description

As per the title

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

File: stories/Collapsible.stories.js [hunk 1]
```diff
@@ -61,3 +61,52 @@ stories.add('Expandable', () => (
...
+    <CollapsibleItem
+      header="Better safe than sorry. That's my motto."
+      icon="filter_drama"
+    >
+      Better safe than sorry. That's my motto.
+    </CollapsibleItem>
+    <CollapsibleItem
+      header="Yeah, you do seem to have a little 'shit creek' action going."
+      icon="place"
+      className="active"
```
Comment (alextrastero) [inline stories/Collapsible.stories.js]: should this be preselected? or how should we tackle this?

Comment (CloudPower97) [reply stories/Collapsible.stories.js]: @alextrastero I'm sorry, I don't grasp what are you trying to say :smile:

Comment (alextrastero) [reply stories/Collapsible.stories.js]: hehe sorry, shouldn't one of these components have an interface to **be** preselected?

Comment (CloudPower97) [reply stories/Collapsible.stories.js]: Sorry but I still don't understand :smile:

Comment (alextrastero) [reply stories/Collapsible.stories.js]: `CollapsibleItem` has a prop (interface) `expanded` which has the same effect of adding the class `active`, let's use that prop best :)

Comment (CloudPower97) [reply stories/Collapsible.stories.js]: Yep, you are right @alextrastero , sorry for the misunderstanding

**Pull request merged**